### DISCUSSION TITLE
Study session: did:element team for an overview and tips

### DISCUSSION THREAD

**Discussion by ottomorac** [veramolabs/did-eth#15] [Ideas]
Hi Team,

One additional study session we should have I think would be with the team from Transmute that implemented the Sidetree protocol into [Element](https://github.com/transmute-industries/sidetree.js). I have the contact of Orie Steele he would probably be able to provide some valuable lessons learned that we can incorporate. 

Thoughts?

**Comment by mirceanis** [comment]
That would be interesting, although these 2 did methods don't share much, besides both being based in ethereum.
Elem is not really on-chain, where as I think one of the interesting features of did:ethr/eth/evm/e is that it is at least partially solving for on-chain uses too
